azurerm/resource_arm_batch_pool.go

```diff
@@ -243,6 +248,8 @@ func resourceArmBatchPoolCreate(d *schema.ResourceData, meta interface{}) error
 	poolName := d.Get("name").(string)
 	displayName := d.Get("display_name").(string)
 	vmSize := d.Get("vm_size").(string)
+	maxTasksPerNode := d.Get("max_tasks_per_node").(int)
```

Comment:
Can we directly assert it as `int32` ?

Reply:
I've removed the temporary variable now!